source/includes/fact-non-utf8-data.rst

```diff
@@ -0,0 +1,34 @@
+Compass can have issues displaying collections if documents have 
+invalid UTF8 characters.
+
+If you attempt to query or export this data, the following error 
+message displays: 
+
+.. code-block:: none
+   :copyable: false
+
+   Invalid UTF-8 string in BSON document. 
+
+In order to query or export this data, you can disable 
+UTF8 validation by setting the ``enableUtf8Validation`` URI option to 
+``false``. 
+
+.. warning::
+
+   **Editing data** with ``enableUtf8Validation=false`` can result in 
+   potential loss of data. This approach is a temporary workaround to 
```

Comment:
(suggestion)
consider "can result in data loss." 
Reasoning is that "can" and "potential" are a bit redundant in this sentence.